pangeo_fish/hmm/filter.py

```diff
@@ -43,6 +45,12 @@ def score(emission, predictor, initial_probability, mask=None):
         prediction = predictor.predict(previous, mask=mask)
         updated = prediction * dask.compute(emission[index, ...])[0]
 
+        if np.sum(updated) == 0:
+            warnings.warn(
+                f"Empty product of the prediction with the true distribution at step {index+1}.",
+                RuntimeWarning,
+            )
+            return 10e5
```

Comment:
this is weird, because it translates to `10 × 10⁵`. Can we instead use the standard syntax:
```suggestion
            return 1e6
```
which translates to `1 × 10⁶`?